data/tutorials/getting-started/2_00_editor_setup.md

```diff
@@ -252,3 +256,28 @@ Add this to your `nvim-lspconfig` setup.
 
 There is no need to pass more settings to `setup` because `nvim-lspconfig` provides reasonable defaults. See [here](https://github.com/neovim/nvim-lspconfig/blob/master/lsp/ocamllsp.lua) for more info.
 
+
+### Installing `ocaml.nvim`
+
+You can install this plugin easily using [lazy.nvim](https://github.com/folke/lazy.nvim) by adding the following to your `ocaml.nvim` setup.
```

Comment:
Is everyone expected to use lazy.nvim ? If not we should also provide standalone installation instructions.